tests/unit/src/utils.spec.ts

```diff
@@ -195,6 +195,44 @@ describe('utils', function () {
       await prepareNpmEnv(cfg, nodeCtx);
       expect(loadSpyOn.mock.calls[loadSpyOn.mock.calls.length - 1]).toMatchSnapshot();
     });
+    it('should configure scoped-registry', async function () {
+      const cfg = _.cloneDeep(runCfg);
+      cfg.npm ||= {};
+      cfg.npm.registries = [{
+        url: 'http://demo.registry.com/npm-test/',
+        scope: '@saucelabs',
+      }];
+      const loadSpyOn = jest.spyOn(npm, 'configure');
+      await prepareNpmEnv(cfg, nodeCtx);
+      expect(loadSpyOn.mock.calls[loadSpyOn.mock.calls.length - 1]).toMatchSnapshot();
```

Comment:
Generally not a huge fan of using `toMatchSnapshot`. Puts the test into change detector test territory, e.g. adding new npm configs will cause the test to fail.